docs/code-examples/README.md

```diff
@@ -9,10 +9,10 @@ Most of these examples are automatically used as docstrings for the `Archetype`
 You can run each example individually using the following:
 
 - **C++**:
-  - `pixi run cpp-build-examples` to compile all examples
-  - `./build/docs/code-examples/<example_name>` to run, e.g. `./build/docs/code-examples/point3d_random`
+  - `pixi run cpp-build-doc-examples` to compile all examples
+  - `./build/docs/code-examples/all/<example_name>` to run, e.g. `./build/docs/code-examples/all/point3d_random`
 - **Python**: `python <example_name>.py`, e.g. `python point3d_random.py`.
-- **Rust**: `cargo r -p code_examples --bin <example_name`, e.g. `cargo r -p code_examples --bin point3d_random`.
+- **Rust**: `cargo run -p code_examples -- <example_name`, e.g. `cargo run -p code_examples -- point3d_random`.
```

Comment:
come to think it's a bit strange that they have different names in all languages!

* cpp: doc examples
* python: example name
* rust: code examples

can we unify this?

Reply:
I'll save that for a follow-up PR